collectors/apps.plugin/apps_plugin.c

```diff
@@ -4173,6 +4173,9 @@ int main(int argc, char **argv) {
         usec_t dt = heartbeat_next(&hb, step);
 #endif
 
+        if (unlikely(write(fileno(stdout), "\n", 1) < 1 && errno == EPIPE))
```

Comment:
I'd prefer a solution like https://stackoverflow.com/questions/9212243/linux-checking-if-a-socket-pipe-is-broken-without-doing-a-read-write instead of calling `write()`.

Reply:
Done.